Formula/k3d.rb

```diff
@@ -22,31 +25,34 @@ class K3d < Formula
   depends_on "go" => :build
 
   def install
+    uri = URI("https://update.k3s.io/v1-release/channels")
+    resp = Net::HTTP.get(uri)
+    resp_json = JSON.parse(resp)
+    k3s_version = resp_json["data"].find { |channel| channel["id"]=="stable" }["latest"].sub("+", "-")
+
     system "go", "build",
            "-mod", "vendor",
-           "-ldflags", "-s -w -X github.com/rancher/k3d/v#{version.major}/version.Version=v#{version}"\
-                       " -X github.com/rancher/k3d/v#{version.major}/version.K3sVersion=latest",
-           "-trimpath", "-o", bin/"k3d"
+           *std_go_args(ldflags: "-s -w -X github.com/rancher/k3d/v#{version.major}/version.Version=#{version} -X github.com/rancher/k3d/v#{version.major}/version.K3sVersion=#{k3s_version}")
 
     # Install bash completion
-    output = Utils.safe_popen_read("#{bin}/k3d", "completion", "bash")
+    output = Utils.safe_popen_read(bin/name, "completion", "bash")
     (bash_completion/"k3d").write output
 
     # Install zsh completion
-    output = Utils.safe_popen_read("#{bin}/k3d", "completion", "zsh")
+    output = Utils.safe_popen_read(bin/name, "completion", "zsh")
     (zsh_completion/"_k3d").write output
 
     # Install fish completion
-    output = Utils.safe_popen_read("#{bin}/k3d", "completion", "fish")
+    output = Utils.safe_popen_read(bin/name, "completion", "fish")
     (fish_completion/"k3d.fish").write output
   end
 
   test do
-    assert_match "k3d version v#{version}\nk3s version latest (default)", shell_output("#{bin}/k3d --version")
+    assert_match "k3d version v#{version}\nk3s version latest (default)", shell_output("#{bin/name} --version")
```

Comment:
Looks like you need to update the assertion here, because it no longer says `version latest`.

Reply:
True, that got reverted with the rest of the original changes in this PR :heavy_check_mark: